feed_integration/app/src/main/java/tv/fw/feedintegration/fragment/FeedViewFragment.kt

```diff
@@ -0,0 +1,48 @@
+package tv.fw.feedintegration.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import tv.fw.common.feed.FeedResource
+import tv.fw.feedintegration.R
+import tv.fw.videofeed.FwVideoFeedView
+import tv.fw.videofeed.baseOptions
+import tv.fw.videofeed.viewOptions
+
+/**
+ * A fragment representing a list of Items.
```

Comment:
❗Is this kinda leftover comment?
❓If no, what do you mean by "a list of items"?

Reply:
Yes, this is a leftover

Reply:
Removed